TombEngine/Game/control/box.cpp

```diff
@@ -574,17 +574,17 @@ void CreatureKill(ItemInfo* creatureItem, int creatureAnimNumber, int playerAnim
 		ItemNewRoom(playerItem.Index, creatureItem->RoomNumber);
 
 	AnimateItem(&playerItem);
-
-	player.ExtraAnim = 1;
+	playerItem.HitPoints = -1;
 	player.Control.HandStatus = HandStatus::Busy;
 	player.Control.Weapon.GunType = LaraWeaponType::None;
+	player.ExtraAnim = 1;
 	player.HitDirection = -1;
 
-	Camera.pos.RoomNumber = playerItem.RoomNumber; 
-	Camera.type = CameraType::Chase;
+	Camera.pos.RoomNumber = playerItem.RoomNumber;
 	Camera.flags = CF_FOLLOW_CENTER;
 	Camera.targetAngle = ANGLE(170.0f);
-	Camera.targetElevation = -ANGLE(25.0f);
+	Camera.targetElevation = ANGLE(-25.0f);
+	Camera.targetDistance = BLOCK(2);
```

Comment:
When Lara dies, the camera moves a bit further to show the general scene, By default is two blocks to simulate the behaviour of the original games, but some bigger creatures like the TRex and the Atlantis mutant moves the camera 3 blocks far. 

(Otherwise, if the camera were close as it normally is, the camera would be moving a lot because the flag makes it aim to Lara bounding box center instead of the transform position).